### PR TITLE
Removed usage of logger.

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Internal: Removed usage of the CKEditor 5 logger. Part of ckeditor/ckeditor5#383.

---

### Additional information

*For example – encountered issues, assumptions you had to make, other affected tickets, etc.*

### CODE REVIEW

File: src/plugincollection.js [hunk 1]
```diff
@@ -189,7 +190,7 @@ export default class PluginCollection {
 			const errorMsg = 'plugincollection-plugin-not-found: Some plugins are not available and could not be loaded.';
 
 			// Log the error so it's more visible on the console. Hopefully, for better DX.
-			log.error( errorMsg, { plugins: missingPlugins } );
+			console.error( attachLinkToDocumentation( errorMsg ), { plugins: missingPlugins } );
```
Comment (pjasiun) [inline src/plugincollection.js:193]: This should be `CKEditorError` exception.

File: src/plugincollection.js [hunk 2]
```diff
@@ -231,7 +232,9 @@ export default class PluginCollection {
 					 * @error plugincollection-load
 					 * @param {String} plugin The name of the plugin that could not be loaded.
 					 */
-					log.error( 'plugincollection-load: It was not possible to load the plugin.', { plugin: PluginConstructor } );
+					console.error( attachLinkToDocumentation(
```
Comment (pjasiun) [inline src/plugincollection.js:235]: This should be `CKEditorError` exception.

File: src/plugincollection.js [hunk 3]
```diff
@@ -371,8 +374,8 @@ export default class PluginCollection {
 			 * @param {Function} plugin1 The first plugin constructor.
 			 * @param {Function} plugin2 The second plugin constructor.
 			 */
-			log.warn(
-				'plugincollection-plugin-name-conflict: Two plugins with the same name were loaded.',
+			console.warn(
```
Comment (pjasiun) [inline src/plugincollection.js]: This should be `CKEditorError` exception.

File: src/plugincollection.js (hunk 1 above)
Comment (ma2ciek) [reply src/plugincollection.js:193]: We throw an error below.

File: src/plugincollection.js (hunk 2 above)
Comment (ma2ciek) [reply src/plugincollection.js:235]: No. We throw the correct error below.

File: src/plugincollection.js (hunk 1 above)
Comment (pjasiun) [reply src/plugincollection.js:193]: :+1:

**Pull request merged**